doc/sphinxext/sphinxgallery/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Sphinx Gallery
 """
 import os
-__version__ = '0.0.8'
+__version__ = '0.0.9-dev'
```

Comment:
Not important for nilearn, but next time you release, you should have in mind to remove the '-dev' tag.